schemas/src/elements/Column.json

```diff
@@ -1,73 +1,73 @@
 {
-  "$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
-  "extends": "ToggleableItem",
-  "description": "Defines a container that is part of a ColumnSet.",
-  "properties": {
-    "items": {
-      "type": "Element[]",
-      "description": "The card elements to render inside the `Column`."
-    },
-    "backgroundImage": {
-      "type": "BackgroundImage",
-      "description": "Specifies the background image. Acceptable formats are PNG, JPEG, and GIF",
-      "version": "1.2",
-      "shorthands": [
-        {
-          "type": "uri-reference",
-          "description": "The URL (or data url) to use as the background image. Supports data URI."
-        }
-      ]
-    },
-    "bleed": {
-      "type": "boolean",
-      "description": "Determines whether the column should bleed through its parent's padding.",
-      "version": "1.2",
-      "features": [
-        2109
-      ]
-    },
-    "fallback": {
-      "type": "Column|FallbackOption",
-      "description": "Describes what to do when an unknown item is encountered or the requires of this or any children can't be met.",
-      "version": "1.2"
-    },
-    "minHeight": {
-      "type": "string",
-      "description": "Specifies the minimum height of the column in pixels, like `\"80px\"`.",
-      "examples": [
-        "50px"
-      ],
-      "version": "1.2",
-      "features": [
-        2293
-      ]
-    },
-    "separator": {
-      "type": "boolean",
-      "description": "When `true`, draw a separating line between this column and the previous column."
-    },
-    "spacing": {
-      "type": "Spacing",
-      "description": "Controls the amount of spacing between this column and the preceding column."
-    },
-    "selectAction": {
-      "type": "ISelectAction",
-      "description": "An Action that will be invoked when the `Column` is tapped or selected. `Action.ShowCard` is not supported.",
-      "version": "1.1"
-    },
-    "style": {
-      "type": "ContainerStyle?",
-      "description": "Style hint for `Column`."
-    },
-    "verticalContentAlignment": {
-      "type": "VerticalContentAlignment",
-      "description": "Defines how the content should be aligned vertically within the column.",
-      "default": "top",
-      "version": "1.1"
-    },
-    "width": {
-      "type": "string|number",
-      "description": "`\"auto\"`, `\"stretch\"`, a number representing relative width of the column in the column group, or in version 1.1 and higher, a specific pixel width, like `\"50px\"`."
-    }
-  }
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"extends": "Element",
```

Comment:
>"extends": "Element" [](start = 1, length = 20)

If columns are not toggleable, it looks like the only thing that extends ToggleableItem is Element. In that case, can we remove ToggleableItem and just add Id and isVisible to Element?